### DISCUSSION TITLE
AlDente 1.19.2

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#819] [General]
* Fixed a bug where Pro users would get a wrong warning message when using Automatic Discharge in Clamshell Mode
* Fixed a number of bugs related to the text field not updating correctly in certain cases
* Fixed a bug in Calibration Mode where AlDente Pro would charge to 100% after the calibration
* Debug file improvements

With peace & love from Vienna,
David & Matthias

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente/releases/tag/1.19.2'>AlDente 1.19.2</a>.</em>

**Comment by ways196** [comment]
Thank you for this amazing app. Is there any way it will be updated to work on Windows as well?

**Reply by MatthiasKerbl** [reply]
Hi @ways196,

Thank you for your kind words.
Do you mean using it in Windows on a MacBook through Bootcamp?

Cheer,
Matthias

**Reply by ways196** [reply]
Thank you for quick reply. No, I mean running it on Windows. Otherwise I have to keep macOS as a secondary system only in order to use AlDente there.

**Reply by MatthiasKerbl** [reply]
AlDente is currently not compatible with Windows. However, if you have an Intel MacBook without Thunderbolt 3, the set Charge Limit in AlDente will automatically persist when you boot into Windows. If you have an Intel MacBook with Thunderbolt 3, you can use the Intel Mode to make the Charge Limit persist when you boot into Windows. You can read more about the Intel Mode here: https://apphousekitchen.com/feature-explanation-intel-mode/
Unfortunately, for both these scenarios you need macOS installed as well.

Cheers,
Matthias

**Reply by ways196** [reply]
Thank you for this information. But is it theoretically possible to write in the value into SMC "BCLM" key while in Windows?

**Reply by MatthiasKerbl** [reply]
Unfortunately, I don't know if it is possible.

Cheers,
Matthias

**Reply by ways196** [reply]
I just thought of a workaround of installing macOS on an external drive and booting from it when needed to change the charge limit value. Thank you.
